dwitter/serializers_v2.py

```diff
@@ -66,6 +66,8 @@ def to_representation(self, instance):
         user = self.context['request'].user
         if user.is_authenticated and user.id == instance.id:
             value['email'] = user.email
+            value['is_staff'] = user.is_staff
+        # 'email' is set to write-only so won't be included for other users
```

Comment:
Not sure if I understand this comment?

Reply:
When looking at that code I was suddenly confused about why email wasn't just returned to all users, and left that comment. I don't think the comment belongs there so it's been removed.

We also have a test now that will break if we start leaking email addresses :)